### PR TITLE
README update to point at weave.works

### PR DESCRIPTION
**Pull request by fintanr**
As mentioned meet us at Dockercon will be removed on the 24th

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 [![Build Status](https://travis-ci.org/weaveworks/weave.svg?branch=master)](https://travis-ci.org/weaveworks/weave) [![Integration Tests](https://circleci.com/gh/weaveworks/weave/tree/master.svg?style=shield&circle-token=4933c7dabb3d0383e62117565cb9d16df7b1a811)](https://circleci.com/gh/weaveworks/weave) [![Coverage Status](https://coveralls.io/repos/weaveworks/weave/badge.svg)](https://coveralls.io/r/weaveworks/weave)
 
+# About Weaveworks
+
+[Come meet Weaveworks at Dockercon](http://weave.works/dockercon)
+
+[Weaveworks](http://weave.works) is the company that develops Weave - the most productive way for developers to connect, observe and control Docker containers. To learn our products, including getting started tutorials, visit our website at [http://weave.works](http://weave.works) or continue on to read about WeaveNet.
```
Comment (bboreham) [inline README.md]: Isn't it 'weave Net' (small first letter and space between words) ?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
 
 [![Build Status](https://travis-ci.org/weaveworks/weave.svg?branch=master)](https://travis-ci.org/weaveworks/weave) [![Integration Tests](https://circleci.com/gh/weaveworks/weave/tree/master.svg?style=shield&circle-token=4933c7dabb3d0383e62117565cb9d16df7b1a811)](https://circleci.com/gh/weaveworks/weave) [![Coverage Status](https://coveralls.io/repos/weaveworks/weave/badge.svg)](https://coveralls.io/r/weaveworks/weave)
 
+# About Weaveworks
+
+[Come meet Weaveworks at Dockercon](http://weave.works/dockercon)
+
+[Weaveworks](http://weave.works) is the company that develops Weave - the most productive way for developers to connect, observe and control Docker containers. To learn our products, including getting started tutorials, visit our website at [http://weave.works](http://weave.works) or continue on to read about WeaveNet.  
+
+# Weave
```
Comment (bboreham) [inline README.md]: Should this be 'Weave Net' ?

**Comment by rade** [comment]
> "To learn our products, including getting started tutorials, visit our website".

"Learn our products"? Did you mean "learn *about*"? But then the "including..." bit doesn't make sense.

**Comment by rade** [comment]
How about: "To learn about our products, and discover how to get started with Weave, visit our website"

**Comment by bboreham** [comment]
Also, you could add a link to the 1.0 blog post, and tell visitors that the Readme material is technical and introductory documentation for some (not all) of weave.

**Comment by fintanr** [comment]
Update to proposed text pushed.

**Pull request merged**